Exygy/askdarcel-web
# Service Listing Accordion

# Pull Request Template

## Issue Overview

This PR addresses - no ticket.

## Description

This ticket adds a new, more accessible accordion element for the ServiceListingPage and deletes the old one. 

## How Can This Be Tested/Reviewed?

Visit any `/services/SERVID` page with multiple addresses i.e. `/services/3640` and do visual and a11y QA on the accordion. Should be able to tab navigate through the top level items and toggle with enter key.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
